### PR TITLE
Setup/store

### PR DESCRIPTION
**Pull request by Jlavarine**
# Description
- This PR will
  - Setup basic app components
  - Use Redux-Toolkit to create API
  - Create a cleaner function for our API function
  - Display art image on main page

## Issues

Closes #

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [ ] Cosmetic changes (changes to layout or visuals in CSS file)

## How Has This Been Tested?

- [x] console.log()
- [x] dev tools
- [x] functionality check through web browser
- [ ] tested by viewing the markdown file preview
- [ ] tested with testing framework (Cypress)

## Checklist:

- [x] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have deleted all comments/console.log() after functionality and understanding is final
- [ ] I have added any issues closed by this PR in the Issues section.
- [x] I have added others as reviewers

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,57 +1,21 @@
...
-import { Counter } from './features/counter/Counter';
+import Nav from './Components/Nav'
+import Container from './Components/Container'
 import './App.css';
 
+import { useGetArtByIdQuery } from './features/currentArt/currentArtAPI'
+
+
 function App() {
+  const id = 144633
```
Comment (whiteheadol) [inline src/App.js:10]: I'm excited to find a way to make this dynamic!

File: src/Components/Art.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+import { useSelector, useDispatch } from 'react-redux'
+import { randomize, selectCurrentArt} from '../features/currentArt/currentArtSlice'
+// import { fetchData } from '../features/currentArt/currentArtAPI'
+
+const Art = () => {
+
+
+
+  const currentArt = useSelector((state) => {
+    const x = state?.artApi?.queries?.['getArtById(144633)']
```
Comment (whiteheadol) [inline src/Components/Art.js:11]: I definitely want to do more reading up on this optional chaining!

File: src/Components/Description.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const Description = () => {
+  return (
+    <div>
+      <p>Title:</p>
```
Comment (nbalmoori) [inline src/Components/Description.js:6]: Let's tackle this in the next pull request!

File: src/features/currentArt/currentArtAPI.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react'
+
+
+// let artObject;
+
+export const artApi = createApi({
```
Comment (nbalmoori) [inline src/features/currentArt/currentArtAPI.js:6]: Really interesting find of how to fix our API fetch call issue

**Pull request merged**